portfolio/src/main/webapp/script.js

```diff
@@ -182,13 +186,26 @@ function constructReplyTree(comment, commentTree, margin) {
   } else {
     let thisReply = createListElement(comment);
     let replyTree = document.createElement("ul");
-    replyTree.className = "comments";
+    replyTree.classList.add("comments", "visiblereplytree");
     const newMargin = margin + 20;
     for (const child of children) {
       const subTree = constructReplyTree(child, commentTree, newMargin);
       replyTree.appendChild(subTree);
     }
     replyTree.style.marginLeft = `${margin}px`;
+    const toggleButton = document.createElement("button");
```

Comment:
This constructReplyTree function is getting a bit long. Separate the creation of toggleButton into its own function?